src/main/java/org/tron/common/runtime/vm/program/Storage.java

```diff
@@ -21,6 +25,15 @@ public Storage(byte[] address, StorageRowStore store) {
     this.store = store;
   }
 
+  public Storage(Storage storage) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Move this left curly brace to the beginning of next line of code. ([squid:LeftCurlyBraceStartLineCheck](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1664&issues=AWbJU6qGSgN4uCclo1dL&open=AWbJU6qGSgN4uCclo1dL)

[ISSUE_KEY]: AWbJU6qGSgN4uCclo1dL